src/Microsoft.AspNet.Identity/IdentityErrorDescriber.cs

```diff
@@ -15,6 +15,15 @@ public virtual IdentityError DefaultError()
                 Description = Resources.DefaultError
             };
         }
+        public virtual IdentityError ConcurrencyFailure()
+        {
+            return new IdentityError
+            {
+                Code = nameof(ConcurrencyFailure),
```

Comment:
I meant that, but only if we end up doing this nameof() thing in most places where we create an IdentityError, in which case I think it would be less error prone.